src/ralph/back_office/models.py

```diff
@@ -58,6 +63,18 @@ def __str__(self):
     def __repr__(self):
         return '<BackOfficeAsset: {}>'.format(self.id)
 
+    def validate_imei(self):
```

Comment:
this in one-field-only related so you could attach it to imei field: https://docs.djangoproject.com/en/1.8/ref/models/fields/#django.db.models.Field.validators